### DISCUSSION TITLE
maintenance dates not updating correctly & dealing with devices not needing maintenance

### DISCUSSION THREAD

**Discussion by BioMedGPH** [Samedis-care/samedis-care-community#59] [Ideas]
I have quite some devices that have no maintenance task planned (but had in the past, and then I deleted because I did not want to include them in the maintenance schedule). But they still show a date in Next Event column in Inventory list. Is this column not updated properly? One example is INF-GPH-MD-765 (but we have many examples, as initially we planned a date for everything, and later decided some things really do not need planned maintenance).

Similarly, I have some devices that had maintenance task completed but show no date in Last Event column. Also here something is not being updated correctly? One example is INF-GPH-MD-353.

Finally, I have quite some devices that are in our inventory, but that we actually don't want to do maintenance for in a planned way. Is there a way to indicate that, so that the Last event and Next event do not show as empty columns. Because I use these columns to check if maintenance is going correctly and not missing out on some devices.

**Comment by YvesR** [comment]
Let me investigate here and checkout how to solve this logical issue and what fits best here.

**Comment by BioMedGPH** [comment]
Maybe there can be a checkbox similar to Is Medical device to select if a device Requires Maintenance (or not)
